examples/keycloak-config-cli-using-spire/keycloak-values.yaml

```diff
@@ -0,0 +1,119 @@
+service:
+  extraPorts:
+    - name: https
+      port: 8443
+      targetPort: 8443
+extraEnvVars:
+  - name: KC_HTTPS_CLIENT_AUTH
+    value: "request"
+  - name: KC_HTTPS_KEY_STORE_FILE
+    value: "/certs/keystore.p12"
+  - name: KC_HTTPS_KEY_STORE_PASSWORD
+    value: "password"
+  - name: KC_HTTPS_KEY_STORE_TYPE
+    value: "pkcs12"
+  - name: KC_HTTPS_TRUST_STORE_FILE
+    value: "/certs/truststore.p12"
+  - name: KC_HTTPS_TRUST_STORE_PASSWORD
+    value: "password"
+  - name: KC_HTTPS_TRUST_STORE_TYPE
+    value: "pkcs12"
+initContainers:
```

Comment:
Instead of ghosttunnel, Could we copy this into the job as well? Would both simplify things and make it more secure.

Reply:
You mean use the `java-spiffe-helper` in conjunction with `keycloak-config-cli` instead of `ghostunnel`?

May work, but beyond my knowledge. `keycloak-config-cli` doesn't mention anything regarding providing your own certificates. Probably would still work by mounting it to some default locations depending on the SSL library being used in Java?

Reply:
I have opened an issue for this some time ago already: https://github.com/adorsys/keycloak-config-cli/issues/959.

Reply:
I see.... hmm.....

What about making the ghosttunnel container a sidecar on the job instead? (initContainer restart=Always), then it could listen only on 127.0.0.1 again to be safe, and wouldn't have its own identity.

Reply:
Yeah, now knowing about this feature explicitly I changed the setup. Should reflect this now. Listens only on `127.0.0.1`.